packages/grafana-prometheus/src/language_provider.ts

```diff
@@ -203,31 +199,25 @@ export default class PromQlLanguageProvider extends LanguageProvider {
     }
   }
 
-  /**
-   * @param key
-   */
-  fetchLabelValues = async (key: string): Promise<string[]> => {
-    const params = this.datasource.getAdjustedInterval(this.timeRange);
+  fetchLabelValues = async (range: TimeRange, key: string): Promise<string[]> => {
```

Comment:
For posterity: I think it's always easier and "more" safe to introduce new parameters as the last parameter. So even functions just calling `fetchLabelValues("__name__")` would behave normally with this new change.
Maybe just something to consider for future changes.